### DISCUSSION TITLE
Logs in whereabouts

### DISCUSSION THREAD

**Discussion by NidhiAdh** [k8snetworkplumbingwg/whereabouts#192] [General]
How to enable debug logs in whereabouts IPAM CNI

**Comment by maiqueb** [comment]
@NidhiAdh you can specify the log location (on the host filesystem) and level via the `log_level` and `log_file` attributes, as [indicated in the docs](https://github.com/k8snetworkplumbingwg/whereabouts/blob/master/doc/extended-configuration.md#logging-parameters).

You can define these in your `net-attach-def`s. Another option would be to provide a cluster-wide configuration, which allows your `net-attach-def`s to be simpler. This setup is also described in the [docs](https://github.com/k8snetworkplumbingwg/whereabouts/blob/master/doc/extended-configuration.md#flatfile-configuration). More details a bit below, on the [example flatfile config](https://github.com/k8snetworkplumbingwg/whereabouts/blob/master/doc/extended-configuration.md#example-flat-file-configuration) section.
